services/ens/contracts.go

```diff
@@ -0,0 +1,63 @@
+package ens
+
+import (
+	"errors"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/status-im/status-go/rpc"
+	"github.com/status-im/status-go/services/ens/registrar"
+	"github.com/status-im/status-go/services/ens/resolver"
+)
+
+var resolversByChainID = map[uint64]common.Address{
+	1: common.HexToAddress("0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e"),
+}
+
+var registrarsByChainID = map[uint64]common.Address{
```

Comment:
```suggestion
var usernameRegistrarsByChainID = map[uint64]common.Address{
```

Reply:
(this is so we know this refers to status username registrar, and not referring to the ens registrars)